must-gather/collection-scripts/gather_ceph_resources

```diff
@@ -191,10 +191,22 @@ for ns in $namespaces; do
         done
     done
 
-    # Add Ready nodes to the list
-    nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers | awk '/\yReady\y/{print $1}')
+    #Add ocs labelled nodes to the list
+    nodes=$(oc get nodes -l cluster.ocs.openshift.io/openshift-storage='' --no-headers|awk '{print $1}')
+
+    #collecting journal logs
+    for node in ${nodes}; do
+    printf "Collecting journal logs from ${node}"
+    oc debug nodes/${node} << CMDS
+       chroot /host
+       cd var
+       journalctl -o verbose >> journal.log &
+       sleep 2
+       exit
+CMDS
```

Comment:
@crombus @obnoxxx @jarrpa 
can you guys please take a look at this indentation?. **Since we have to get inside the nodes and fire these commands** I had no other way apart from this?. If this indentation is correct and makes sense, then it's fine, otherwise, do you guys have any other way to debug into the node and fire multiple commands?.

Reply:
for me indentation is a issue(we can see what can be done). Like you are creating a debug and exiting that cause debug pod to terminate. So I will suggest you to move this clusterscoped file and collect the data in clusteredscope dir.

Reply:
That makes sense.
Let's wait for others and see if there's an even better way to do it.

Reply:
+1

Reply:
Done, please review!.